app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +21,26 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag 
+    {{if (and (not @item.args.positional) @item.isComponent) "component-tree-item__bracket"}} 
+    {{if (and (not @item.args.positional) (not @item.hasChildren)) "component-tree-item__bracket--self-closing"}}
+    {{if (and @item.args.positional @item.isComponent) "component-tree-item-classic__bracket"}} 
+    {{if (and @item.args.positional (not @item.hasChildren)) "component-tree-item-classic__bracket--self-closing"}}"
   >
     {{#if @item.isComponent}}
-      {{classify @item.name}}
+      {{#if @item.args.positional}}
+        {{dasherize @item.name}}
+      {{else}}
+        {{classify @item.name}}
+      {{/if}}
+
+      {{#each-in @item.args.positional as |name value|}}
+        {{component-argument value}}
+      {{/each-in}}
+
+      {{#each-in @item.args.named as |name value|}}
+        @{{name}}={{component-argument value}}
```

Comment:
Need something like this:

```suggestion
        {{#if @item.isCurlyInvocation}}
          {{name}}={{component-argument value}}
        {{else if (is-string value)}}
          @{{name}}={{component-argument value}}
        {{else}}
          @{{name}}=\{{{{component-argument value}}}}
        {{/if}}
```

Reply:
What does this accomplish? The arguments are handled via the `component-argument` helper.

Reply:
If using curly braces, you don't need `@` in front of args or curlies around args. If using angle bracket, you need both except in the string case where you don't need the curlies.

```
{{foo-bar some="arg" another=123456}}
<FooBar @some="arg" @another={{123456}} />
```